components/html-block/demo/html-block.html

```diff
@@ -507,7 +509,7 @@ <h2>HTML Block (LaTeX inline math)</h2>
 						<div>
 							An equation rendered using LaTeX...
 							\( f(x) = \int \mathrm{e}^{-x}\,\mathrm{d}x \)
-							... and some text!
+							... and some text ...
```

Comment:
Oops. Copy/paste error. I'm not going to bother changing it back though; hardly seems worth the CI run-time.